css-canvas-medical/styles.css

```diff
@@ -0,0 +1,215 @@
+.container {
+  width: 85%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.justify-space-between {
+  justify-content: space-between;
+}
+
+.color-6E6E6E {
+  color: #6e6e6e;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.text-align-center {
+  text-align: center;
+}
+
+.margin-left-right-auto {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.margin-bottom-75 {
+  margin-bottom: 75px;
+}
+
+.margin-top-40 {
+  margin-top: 40px;
+}
+
+.column-half {
+  width: 50%;
+}
+
+.border-bottom {
+  border-bottom: 1px solid #e6e9ec;
+  width: 100%;
+}
+
+.padding-75 {
+  padding: 75px;
+}
+
+.padding-top-bottom-15 {
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.padding-top-bottom-50 {
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.padding-top-bottom-100 {
+  padding-top: 100px;
+  padding-bottom: 100px;
+}
+
+.flex-end {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  align-items: center;
+  color: #adadad;
+}
+
+.bg-FAFAFA {
+  background-color: #fafafa;
+}
+
+.bg-515351 {
+  background-color: #515351;
+  color: white;
+}
+
+.line-height-40 {
+  line-height: 40px;
+}
+
+.line-height-10 {
+  line-height: 10px;
+}
+
+button {
+  border-radius: 5px;
+  font-weight: 400;
+}
+
+a,
+img[src="images/logo-blue.svg"],
```

Comment:
Instead of this complicated selector for the `img`, you should just add a class to it.

Reply:
This still needs to be addressed